images/linux/scripts/installers/cmake.sh

```diff
@@ -11,7 +11,7 @@ if command -v cmake; then
 else
 	json=$(curl -s "https://api.github.com/repos/Kitware/CMake/releases")
 	latest_tag=$(echo $json | jq -r '.[] | select(.prerelease==false).tag_name' | sort --unique --version-sort | tail -1)
-	sh_url=$(echo $json | jq -r ".[] | select(.tag_name==\"${latest_tag}\").assets[].browser_download_url | select(endswith(\"Linux-x86_64.sh\"))")
+	sh_url=$(echo $json | jq -r ".[] | select(.tag_name==\"${latest_tag}\").assets[].browser_download_url | select(endswith(\"linux-x86_64.sh\"))")
```

Comment:
Could you please remove letter `l or L` to parse both uri variants?
`linux-x86_64.sh -> linux-x86_64.sh`

Reply:
nice suggestion! I will change `inux-x86_64.sh`

Reply:
I changed in https://github.com/actions/virtual-environments/pull/2731/commits/04e385fe38e8f3521715ffecaba5f32065833d23